packages/node-opentelemetry/src/otel.ts

```diff
@@ -18,11 +19,24 @@ const LOG_PREFIX = `⚠️  ${_LOG_PREFIX}`;
 
 const env = process.env;
 
+const customInstrumentationMap = {
+  '@opentelemetry/instrumentation-remix': RemixInstrumentation,
+};
+
+type ConfigArg<T> = T extends new (...args: infer U) => unknown ? U[0] : never;
+export type CustomInstrumentationConfigMap = {
+  [Name in keyof typeof customInstrumentationMap]?: ConfigArg<
+    (typeof customInstrumentationMap)[Name]
+  >;
+};
+
 type SDKConfig = {
   advancedNetworkCapture?: boolean;
   betaMode?: boolean;
   consoleCapture?: boolean;
   instrumentations?: InstrumentationConfigMap;
+  additionalInstrumentations?: CustomInstrumentationConfigMap[];
+  disableInstrumentations?: string[];
```

Comment:
I think users should be able to disable existing instrumentations using `instrumentations` arg. we can remove this for now

Reply:
I was looking at that but I wasnt too sure how a user would be able to do that themselves, but if they can just recreate the instrumentations arg that makes sense. Just to make sure I got it, they would be able to just add that code snippet to initSDK config parameters?

Reply:
Correct, they can add that code snippet to `instrumentations` arg to disable default instrumentation pkgs

Reply:
Ah okay that makes a lot more sense than what I was trying to do haha!